packages/backend/src/handlers/remote-schemas/types/uuid.ts

```diff
@@ -0,0 +1,38 @@
+import { GraphQLScalarType } from 'graphql';
+import { Kind } from 'graphql/language';
+
+const uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+const nilUUID = '00000000-0000-0000-0000-000000000000';
```

Comment:
When would this be the case? that this is the value

Reply:
this is just the nilUUID which is also part of the UUID space but not covered by the regex.